tessera-core/src/main/java/com/quorum/tessera/node/PartyInfoServiceImpl.java

```diff
@@ -34,7 +34,7 @@ public PartyInfoServiceImpl(final PartyInfoStore partyInfoStore,
         this.partyInfoStore = Objects.requireNonNull(partyInfoStore);
         this.configService = Objects.requireNonNull(configService);
 
-        final String advertisedUrl = configService.getServerUri().toString() + "/";
```

Comment:
Probably need a better way to do this, but we need to make sure the advertised URL ends with a "/" for backwards compatibility.
Maybe doing a check on the string to see if it ends with a slash, and if not, adding it? Instead of blindly adding the trailing slash (although the way we create the URI should mean it will always be needed).